CMakeLists.txt

```diff
@@ -52,10 +56,19 @@ ADD_LIBRARY(graphqlparser SHARED
   lexer.h
   GraphQLParser.cpp)
 
-# Enable this and remove CMAKE_CXX_FLAGS fiddle above when we are able
-# to upgrade to CMake 2.8.12. Blocker seems to be Travis CI being on
-# Ubuntu Precise; Trusty has 2.8.12.
-# TARGET_COMPILE_OPTIONS(graphqlparser PUBLIC -std=gnu++11)
+IF(UNIX)
+  SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=gnu++11")
+
+  ADD_LIBRARY(graphqlparser SHARED ${GRAPHQLPARSER_SOURCES})
+
+  # Enable this and remove CMAKE_CXX_FLAGS fiddle above when we are able
+  # to upgrade to CMake 2.8.12. Blocker seems to be Travis CI being on
+  # Ubuntu Precise; Trusty has 2.8.12.
+  # TARGET_COMPILE_OPTIONS(graphqlparser PUBLIC -std=gnu++11)
+ELSEIF(WIN32)
+  ADD_LIBRARY(graphqlparser STATIC ${GRAPHQLPARSER_SOURCES})
```

Comment:
Actually, I just found a blog [post](https://blog.kitware.com/create-dlls-on-windows-without-declspec-using-new-cmake-export-all-feature/) describing how to do that automatically. Initial testing with `dump_json_ast` linked against the DLL instead of the static lib seems to work.